pkg/notes/internal/template.go

```diff
@@ -0,0 +1,85 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package internal
+
+// DefaultReleaseNotesTemplate is the text template for the default release notes.
+// k8s/release/cmd/release-notes uses text/template to render markdown
+// templates.
+
+var DefaultReleaseNotesTemplate = `
```

Comment:
Do we want to make it `const` and private?

Reply:
We can make it const but we can't make it private because:
* It's used in the `options` package and it's test. I think we need it there so we can have a default value for "--format=go-template"

It's also in the notes/internal package so I think by definition it can't be imported outside of notes if you're concerned about leaking.

Reply:
SGTM.

Reply:
On second thought. I made it const and private. I decided to move to lazy template reading/parsing because it makes the logic in main simpler.